schema.graphql

```diff
@@ -132,13 +135,36 @@ type Shipment {
   sendingHub: Group!
   receivingHubId: Int!
   receivingHub: Group!
+  pricing: ShipmentPricing
 
   # default to created
   statusChangeTime: Date!
   createdAt: Date!
   updatedAt: Date!
 }
 
+type ShipmentPricing {
+  singlePallet: MoneyAmount
+  halfPallet: MoneyAmount
+}
+
+input ShipmentPricingInput {
+  singlePallet: MoneyAmountInput
+  halfPallet: MoneyAmountInput
+}
+
+type MoneyAmount {
```

Comment:
maybe this should just be called `Money`?

Reply:
I think `MoneyAmount` describes it better